source/controllers/SectionController.ts

```diff
@@ -7,30 +7,40 @@ import { headerConstants } from './config.js';
 const section = new Controller('section');
 
 section.read = async (req, res) => {
+	if (isLoggedOn(res.locals.user)) {
+		if (res.locals.user.banned) {
+			res.clearCookie('AuthToken');
+			res.redirect('/section');
+			return;
+		}
+	}
 	const sections = await Section.findAll();
 	res.render('pages/main/sections_mainpage.ejs', {
 		sections,
 		sectionName: section.name,
-		constants: headerConstants
+		constants: headerConstants,
+		originURL: '_section' // For redirect purposes with login button
 	});
 };
 
 interface SectionCreateForm {
 	section_name: string;
+	lv_section_name: string;
 };
 
 interface SectionEditForm {
 	section_name: string;
+	lv_section_name: string;
 	section_id: string;
 };
 
 function ValidSectionCreateForm (obj: any): obj is SectionCreateForm {
-	const valid = ('section_name' in obj) && typeof obj.section_name === 'string';
+	const valid = ('section_name' in obj) && typeof obj.section_name === 'string' && ('lv_section_name' in obj) && typeof obj.lv_section_name === 'string';
```

Comment:
This is a really long statement. I think it should be split into multiple lines.